src/shared/frame-rpc.js

```diff
@@ -53,46 +60,67 @@ const VERSION = '1.0.0';
 export class RPC {
   /**
    * Create an RPC client for sending RPC requests from `sourceFrame` to
-   * `destFrame`.
+   * `destFrame`, and receiving RPC responses from `destFrame` to `sourceFrame`.
    *
-   * @param {Window} sourceFrame
-   * @param {Window} destFrame
-   * @param {string} origin - Origin of destination frame
-   * @param {Record<string, (...args: any[]) => any>} methods - Map of method
+   * This class has been adapted to work with `MessageChannel`. Because
+   * `MessageChannel` are unidirectional we ignore the `sourceFrame` and `origin`
+   * properties, in this case.
+   *
+   * TODO: July 2021, currently this class is a bit of a _frankenstein_ because
+   * we support both, `Window.postMessage` and `MessagePort.postMessage`.
```

Comment:
```suggestion
   * we support both `Window.postMessage` and `MessagePort.postMessage`.
```

I'm being ultra-pedantic.